macast/macast.py

```diff
@@ -410,6 +410,10 @@ def build_setting_menu(self):
                                                       _("Pattern"),
                                                   ], self.on_menubar_icon_change_click))
         else:
+            if not sys.executable.split('\\')[-1] == 'python.exe':
```

Comment:
The Linux platform is ignored, and there should be a better way to judge.
like:
```python
if "python" in os.path.basename(sys.executable).lower()
```